src/components/donation/Steps/CurrentStep.tsx

```diff
@@ -12,34 +12,34 @@ export default function CurrentStep({ config }: Props) {
   const state = useGetter((state) => state.donation);
   const dispatch = useSetter();
 
+  if (state.step === "init") return <></>; // <Steps /> sets to step 1 onMount
```

Comment:
@SovereignAndrey unrelated to the current PR, but I really think when a user gets to the "Thanks for donating" page, there should be a button to go back to the page of the endowment to which the user just donated instead of "going back to the platform" -> the user is *already* on the platform, if they wished to go back they would probably want to go to the endowment's profile and furthermore, if they wished to go to the Marketplace, they can just click the appropriate button in the navbar (so no need for an additional one on the page).
There were many times when I needed such a button while working on donation flows, but instead I had to manually input the URL in the navbar (much faster than going through Marketplace). 
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/0834b3a2-ca51-412d-bf9e-099a63190701)

Reply:
@ap-justin This is looking good overall! Lot of little things here and there that can be improved to get further inline style and UI wise, but this is a great base to work off of for future (smaller) PRs and shouldn't be a blocker to moving forward. 

Re: @misicnenad's comments about syncing up the payment methods, I agree it jumped out at me too when reviewing the UI. To that end, I think we can incorporate a "Back" link and "Cancel" link at the top of the section/page, much like every.org does. This would mean there's only one button at the bottom that moves the user forward, "Continue"/"Pay"/etc. It probably helps with donation conversions to keep users from dropping off as readily when the back and cancel are not _quite_ easy to get to, but there nonetheless.

Here's the screenshot from their site (having clicked into the Venmo section's flow by at least one step):
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/85138450/973784b6-4351-478a-b22a-5b341530e55d)

One important distinction that we'll need to consider is the "Cancel" link placement outside the middle area. Since the widget will use this UI too, we'll need to make sure we consider that the widget view will NOT have the flanking side bars (unlike the official donation page...eventually) and we'll need to move that over closer to the middle. Our view will thus be like their mobile view I think regardless of desktop/mobile: 
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/85138450/dbe70982-a1e8-4ad5-81ee-9907238e36fc)

If we move it down a bit that might be the ticket:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/85138450/9e8fd1ee-7a9c-4669-897e-2912dad50af2)

Reply:
>  there should be a button to go back to the page of the endowment to which the user just donated instead of "going back to the platform" -> the user is already on the platform, if they wished to go back they would probably want to go to the endowment's profile and furthermore, if they wished to go to the Marketplace, they can just click the appropriate button in the navbar (so no need for an additional one on the page).

@misicnenad Fair point! Though I'd argue that if a user has just finished donating to a nonprofit they are less likely to go back to their profile page and instead, as I think the original intent was for this link, can be encouraged to go donate more (ie. back to the Marketplace to find new nonprofits).

Reply:
@SovereignAndrey 
> One important distinction that we'll need to consider is the "Cancel" link placement outside the middle area. Since the widget will use this UI too, we'll need to make sure we consider that the widget view will NOT have the flanking side bars (unlike the official donation page...eventually) and we'll need to move that over closer to the middle. Our view will thus be like their mobile view I think regardless of desktop/mobile:

got this ser, may not be a problem since "cancel" is just a link to profile and would now be owned by the container of multistep form. 

<img width="453" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/33d6a553-03c8-47df-8d3b-91be45cc4f76">

For the `/donate-page` - just how it is handled by every org (at the top). For the widget, may not be needed as currently, we don't even have a "cancel" button. 

The title could even be a link if there's a need for widget donor to go to the original profile page 
<img width="822" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/24718427-dd1c-4429-a4ea-e0c30778c8ec">

Reply:
@ap-justin Some other smaller changes needed are (example in screenshot):
- [ ] the use of icons next to payment options
- [ ] collapse/show more options when in space restricted mobile view (no more than 4 buttons / two rows) 

![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/85138450/93048272-c1c5-4b76-b7a6-1788278e5e37)

Reply:
>For the /donate-page - just how it is handled by every org (at the top). For the widget, may not be needed as currently, we don't even have a "cancel" button.
> [sic]
> For the /donate-page - just how it is handled by every org (at the top). For the widget, may not be needed as currently, we don't even have a "cancel" button.

@ap-justin All good regarding the above. :+1: 
No need to link to original profile from widget for now, though that could be something to consider doing from the widget "Thank You" page to bring users to the main app.

Reply:
@SovereignAndrey , applied additional requirements (icons, and more options)
<img width="473" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/303c5c82-9fd1-4579-ac7f-5cab045d8138">

<img width="223" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/8917b87b-d526-4a3e-b360-bcabb580ac73">